command/auth_enable.go

```diff
@@ -96,6 +99,20 @@ func (c *AuthEnableCommand) Flags() *FlagSets {
 			"TTL.",
 	})
 
+	f.StringSliceVar(&StringSliceVar{
+		Name:   flagNameAuditNonHMACRequestKeys,
+		Target: &c.flagAuditNonHMACRequestKeys,
+		Usage: "Comma-separated list of keys that will not be HMAC'd by audit" +
```

Comment:
`Comma-separated string or list of keys...` here and for all similar references below.